### ISSUE TITLE
Choppy animations

### ISSUE THREAD

**Issue by cpsievert** [tdhock/animint#54]
To reduce the initial choppiness of animation like [this](http://cpsievert.github.io/baseballR/20150415/mark/), it would be nice to have a programmatic way to pause the animation by default (so the user can wait until all the necessary data is loaded before starting the animation). 

A better solution might be to dynamically pause the animation until a certain number of chunks have been loaded. 

Note that I've tried the `chunk_vars=character()` trick to prevent data from being split into chunks, but I received this error:

![screen shot 2015-04-14 at 2 50 29 pm](https://cloud.githubusercontent.com/assets/1365941/7256085/e0f6b924-e812-11e4-8096-07dc40df8d6b.png)

**Comment by tdhock** [comment]
Can you post the source code for your viz so we can use it as a testing example?

It should be quite easy to modify the renderer to pause the animation until all time-related data have been loaded.

**Comment by cpsievert** [comment]
The example is a bit complicated, so I'll come up with an easier one and add it as a test.

**Comment by cpsievert** [comment]
You're on a roll @tdhock! :+1:
